authentication/auth_method_oidc.go

```diff
@@ -65,6 +65,7 @@ func (a oidcAuth) getMSALToken(ctx context.Context, api environments.Api, _ auto
 func (a oidcAuth) populateConfig(c *Config) error {
 	c.AuthenticatedAsAServicePrincipal = true
 	c.AuthenticatedViaOIDC = true
+	c.UseMicrosoftGraph = true
```

Comment:
I think we can remove this in favour of adding it to the `isApplicable` function above?